FogCreek/Glitch-Community
# Consistent Community 404 Illustration

## Links
* Remix link: https://community-404.glitch.me
* Issue-tracker link: https://github.com/FogCreek/Glitch-Community-Work/issues/249

## GIF/Screenshots:
Replaced compass illustrations + telescope with empty aquarium animation for more consistent 'not found' UX

![aquarium](https://user-images.githubusercontent.com/519336/62236144-79b9ca80-b39c-11e9-9be2-fb1c2c8007ea.gif)

## Changes:
* Page Not Found page (https://community-404.glitch.me/awegoik)
![image](https://user-images.githubusercontent.com/519336/62236314-db7a3480-b39c-11e9-8135-ba17e08ef11f.png)

* Filtering on user profiles:
![image](https://user-images.githubusercontent.com/519336/62236327-e03ee880-b39c-11e9-8cc1-f51e2975a058.png)

* No search results:
![image](https://user-images.githubusercontent.com/519336/62236345-e634c980-b39c-11e9-8c21-ebe423f26bde.png)



## How To Test:
* Check the 404 page (https://community-404.glitch.me/awegoik)
* Go to a user profile page (https://community-404.glitch.me/@scientiffic) and try filtering collections or projects by name
* Search for something that doesn't have search results (https://community-404.glitch.me/@scientiffic)
